src/py/flwr/cli/new/new.py

```diff
@@ -140,6 +141,7 @@ def new(
     # Depending on the framework, generate task.py file
     frameworks_with_tasks = [
         MlFramework.PYTORCH.value.lower(),
+        MlFramework.HUGGINGFACE.value.lower(),
```

Comment:
Could you explain why this needs to be here? I don't quite get the purpose of this python file and the top-level docs doesn't help much :(

Reply:
@adam-narozniak The `task.py` file contains most of the "non-flower dependent" functions needed to perform training and evaluation. Here, we list the frameworks that contain a `task.py` file in their template. Note that this approach might change in the future, as we might create a template folder for each framework, instead of assigning the template files based on the filenames

Reply:
Thanks for the explanation. It makes sense.